src/objects/ConversationManager.js

```diff
@@ -38,14 +38,56 @@ ConversationManager.prototype.getCurrentText = function () {
   return this.conversation[this.idx]['text'];
 };
 
-ConversationManager.prototype.getResponses = function () {
+ConversationManager.prototype.getResponses = function (game) {
```

Comment:
Should this (and getCondition) be using its own this._game object, if that's going to be our precedent going forward?